### ISSUE TITLE
Qualcomm Atheros Killer E2500 Gigabit

### ISSUE THREAD

**Issue by GitNaufal** [dortania/bugtracker#175]
### Link to pages with the issue
https://dortania.github.io/OpenCore-Install-Guide/ktext.html#ethernet

---

### Explaination of the issue
I don't think all E2500 models are Realtek based.
Mine doesn't work when using RealtekRTL8111.kext, otherwise when using AtherosE2200Ethernet.kext do work like a charm

![Screen Shot 2021-03-15 at 14 51 43](https://user-images.githubusercontent.com/80172679/111120497-fd3e2480-859d-11eb-9a0e-a34244b2f40e.png)
![Screen Shot 2021-03-15 at 14 51 27](https://user-images.githubusercontent.com/80172679/111120505-ff07e800-859d-11eb-8013-d23bd2c876a5.png)

**Comment by dreamwhite** [comment]
In this case, I'd suggest to correct the following note statement with:
![image](https://user-images.githubusercontent.com/17933708/111132605-d8d05100-8579-11eb-9fb9-2e1268907fb7.png)

`Note: some Atheros Killer E2500 models may be Realtek based. For these systems if AtherosE2200Ethernet is not working please use RealtekRTL8111 instead`

CC @khronokernel do you agree?

**Comment by GitNaufal** [comment]
> In this case, I'd suggest to correct the following note statement with:
> ![image](https://user-images.githubusercontent.com/17933708/111132605-d8d05100-8579-11eb-9fb9-2e1268907fb7.png)
> 
> `Note: some Atheros Killer E2500 models may be Realtek based. For these systems if AtherosE2200Ethernet is not working please use RealtekRTL8111 instead`
> 
> CC @khronokernel do you agree?

right, it is.
thx for mention the statement.

**Comment by lingenfr** [comment]
Unless someone can verify that "any" E2500 is Realtek based, I suggest you remove that statement. If so, I suggest you reword it to say, "If you've tried the E2200 kext and it didn't work, your E2500 may be Realtek based. Try RealtekRTL8111 instead." I've done a lot of searching and not found one post that indicated the Realtek kext worked with the E2500.
